src/main/java/de/hpi/bpt/argos/storage/hierarchy/HierarchyBuilder.java

```diff
@@ -0,0 +1,14 @@
+package de.hpi.bpt.argos.storage.hierarchy;
+
+/**
+ * This interface offers methods to build the entityHierarchy.
+ */
+@FunctionalInterface
+public interface HierarchyBuilder {
+
+	/**
+	 * This method returns the root node of the entityHierarchy.
+	 * @return - the root node of the entityHierarchy
+	 */
+	EntityHierarchyNode getEntityHierarchy();
```

Comment:
I chose this name since all you gonna know about the hierarchy is the root node. Therefore I liked the semantics of "getting the whole hierarchy" better.